src/components/toggle/Toggle.scss

```diff
@@ -15,7 +15,7 @@
     }
 }
 
-.toggle {
+.bdl-Toggle {
```

Comment:
Should the child classes like `toggle-btn` also be updated?

Reply:
I can't find any instances of `.toggle` in BUIE, EUA, Dev Console, or Admin Console. Should we remove this class? The `Toggle` component uses `.toggle-container`, which seems like a better candidate for `.bdl-Toggle`.